nano/secure/common.hpp

```diff
@@ -626,6 +626,7 @@ class election_status final
 {
 public:
 	election_status ();
+	election_status (rsnano::ElectionStatusHandle const * handle);
```

Comment:
The point should not be const because election_status should take ownership of the handle